zigConsole/.vscode/launch.json

```diff
@@ -0,0 +1,211 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Local AMD64",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/zig-deploy/x86_64-linux-gnu/debug/bin/__change__",
+            "args": [],
+            "stopAtEntry": false,
+            "cwd": "${workspaceRoot}",
+            "environment": [],
+            "externalConsole": false,
+            "MIMode": "gdb",
+            "miDebuggerPath": "/usr/bin/gdb",
+            "setupCommands": [
+                {
+                    "description": "Enable pretty-printing for gdb",
+                    "text": "-enable-pretty-printing",
+                    "ignoreFailures": true
+                },
+                {
+                    "description":  "Set Disassembly Flavor to Intel",
+                    "text": "-gdb-set disassembly-flavor intel",
+                    "ignoreFailures": true
+                }
+            ],
+            "preLaunchTask": "build-debug-amd64-local"
```

Comment:
this tasks appears to not be present, I check and there is the `build-debug-amd64`

Reply:
Based on C and C++ solution

https://github.com/toradex/vscode-torizon-templates/blob/bookworm/cConsole/.vscode/launch.json#L28